pkg/config/config.go

```diff
@@ -92,8 +86,9 @@ func (n GroupName) Validate() error {
 	if n == "" {
 		return errors.New(errorMessages["emptyGroupName"])
 	}
-	if hasIllegalChars(string(n)) {
-		return fmt.Errorf("%s %s", errorMessages["illegalChars"], n)
+
+	if !regexp.MustCompile(`^[\w]+[\w-]*$`).MatchString(string(n)) {
```

Comment:
I think it might be wise to disallow ending in a dash. WDYT?

Reply:
Sounds right, though I can't formulate why ?)

Reply:
Probably harder to detect as a character vs operator? In any case, it's a rule with GCP resource naming.

https://cloud.google.com/compute/docs/naming-resources#resource-name-format